cmd/gosec/main.go

```diff
@@ -380,13 +383,24 @@ func main() {
 
 	excludedDirs := gosec.ExcludedDirsRegExp(flagDirsExclude)
 	var packages []string
-	for _, path := range flag.Args() {
+
+	if len(flag.Args()) == 0 {
+		path := "./..."
 		pcks, err := gosec.PackagePaths(path, excludedDirs)
 		if err != nil {
 			logger.Fatal(err)
 		}
 		packages = append(packages, pcks...)
+	} else {
+		for _, path := range flag.Args() {
```

Comment:
I would avoid this code duplication here. You can do something like:
```Go
paths := flag.Args()
if len(paths) == 0 {
  paths = append(paths, "./...")
}
for _, path := range paths {
  ...
}
```

Reply:
You're right, I'll fix that :)